engine/classes/ElggFile.php

```diff
@@ -294,12 +294,13 @@ public function close() {
 	/**
 	 * Delete this file.
 	 *
+	 * @param bool $follow_symlinks If true, will also delete the target file if the current file is a symlink
 	 * @return bool
 	 */
-	public function delete() {
+	public function delete($follow_symlinks = true) {
```

Comment:
I thought about that, but looks like not.
On Apr 9, 2016 9:20 PM, "Steve Clay" <notifications@github.com> wrote:

> In engine/classes/ElggFile.php
> <https://github.com/Elgg/Elgg/pull/9647#discussion_r59119959>:
>
> >  	 * @return bool
> >  	 */
> > -	public function delete() {
> > +	public function delete($follow_symlinks = true) {
>
> We don't get an E_STRICT error here, from not matching ElggEntity?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9647/files/facc13fe13f8719204c85b7f7334acfa5e8f06da#r59119959>
>